sdk/communication/Azure.Communication.CallingServer/tests/CallConnection/CallConnectionLiveTests.cs

```diff
@@ -401,38 +351,53 @@ public async Task RunCreateAddCreateAudioRoutingRemoveHangupScenarioTests()
+                Assert.IsTrue(getAudioGroupgResult.AudioRoutingMode == AudioRoutingMode.Multicast);
+                Assert.NotNull(getAudioGroupgResult);
 
                 // Add Another Participant
                 AddParticipantResult addAnotherParticipantResult = await AddParticipantOperation(callConnection, anotherUserId).ConfigureAwait(false);
                 Assert.NotNull(addAnotherParticipantResult);
                 await WaitForOperationCompletion().ConfigureAwait(false);
 
+                // Add Another Participant To Default Audio Group
+                var addAnotherParticipantToDefaultAudioGroupResult = await AddParticipantToDefaultAudioGroupOperation(callConnection, anotherUserId).ConfigureAwait(false);
+                Assert.NotNull(addAnotherParticipantToDefaultAudioGroupResult);
+                await WaitForOperationCompletion().ConfigureAwait(false);
+
                 // Creating Another List And Adding Participant
                 List<CommunicationUserIdentifier> participantsList = new List<CommunicationUserIdentifier>();
                 participantsList.Add(new CommunicationUserIdentifier(anotherUserId));
 
-                // Update Audio Routing Group
-                await UpdateAudioRoutingGroupOperation(callConnection, audioRoutingGroupId, participantsList).ConfigureAwait(false);
+                // Update Audio Group
+                await UpdateAudioGroupOperation(callConnection, audioGroupId, participantsList).ConfigureAwait(false);
+                // Get Audio Group
+                var getUpdatedAudioGroupResult = await GetAudioGroupOperation(callConnection, audioGroupId).ConfigureAwait(false);
+                Assert.IsTrue(getUpdatedAudioGroupResult.AudioRoutingMode == AudioRoutingMode.Multicast);
+                Assert.NotNull(getUpdatedAudioGroupResult);
+
+                // Delete Audio Group
+                await DeleteAudioGroupOperation(callConnection, audioGroupId).ConfigureAwait(false);
 
-                // Get Audio Routing Group
-                var getUpdatedAudioRoutingResult = await GetAudioRoutingGroupOperation(callConnection, audioRoutingGroupId).ConfigureAwait(false);
-                Assert.IsTrue(getUpdatedAudioRoutingResult.AudioRoutingMode == AudioRoutingMode.Multicast);
-                Assert.NotNull(getAudioRoutingResult);
+                // Remove First Added Participant from Default Audio Group
+                await RemoveParticipantFromDefaultAudioGroupOperation(callConnection, userId).ConfigureAwait(false);
 
-                // Delete Audio Routing Group
-                await DeleteAudioRoutingGroupOperation(callConnection, audioRoutingGroupId).ConfigureAwait(false);
+                // Remove Another Participant from Default Audio Group
+                await RemoveParticipantFromDefaultAudioGroupOperation(callConnection, anotherUserId).ConfigureAwait(false);
```

Comment:
let us not add this to this test. this test should only handle the renaming of AudioRoutingGroup to AudioGroup for Create.Get.Update,Remove